Tests/PackageGraphTests/ModulesGraphTests.swift

```diff
@@ -114,6 +114,7 @@ struct ModulesGraphTests {
 
     @Test(
         .IssueWindowsPathTestsFailures,
+        .disabled("Requires swift-testing bug fix https://github.com/swiftlang/swift-testing/pull/1441"),
```

Comment:
question: Do we really need to disabled this test?  Can we instead guard this based on compiler version?  For example, see the `requireSwift6_2` trait

https://github.com/swiftlang/swift-package-manager/blob/main/Sources/_InternalTestSupport/SwiftTesting%2BTraitConditional.swift#L200-L208